function.tf

```diff
@@ -160,3 +160,61 @@ resource "google_cloud_scheduler_job" "this" {
     }
   }
 }
+
+resource "google_cloudfunctions_function" "rest_of_assets" {
+  count = var.enable_function ? 1 : 0
+
+  name                  = "${var.name}_rest_of_assets"
```

Comment:
is `var.name` going to be unique when customers install it? (i expect yes, but just want to check)

Reply:
The module granularly this can be deployed is at the project level and the function names can be duplicated across projects so this should be fine (we've tested deploying it to multiple projects and haven't hit any issues).